.eslintignore

```diff
@@ -6,3 +6,4 @@ coverage/*
 keys/*
 lib/*
 packages/reputation-miner/node_modules/*
+packages/colony-contract-loader-network/node_modules/*
```

Comment:
It might be good to do a
packages/**/node_modules here

Reply:
That is a better idea.